flogin/plugin.py

```
@@ -89,6 +90,48 @@ def __init__(self, **options: Any) -> None:
+    @cached_property
+    def flow_version(self) -> str:
+        """:class:`str`: the flow version from environment variables.
+
+        .. versionadded:: 1.0.1
+        """
+
+        try:
+            return os.environ["FLOW_VERSION"]
+        except KeyError:
+            raise RuntimeError(
+                f"The 'FLOW_VERSION' environment variable is not set"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)